e2e/src/main/java/org/jetbrains/bsp/bazel/base/BazelBspTestBaseScenario.java

```diff
@@ -60,7 +60,11 @@ public void executeScenario() {
   }
 
   private boolean executeScenarioSteps() {
-    return getScenarioSteps().stream().allMatch(BazelBspTestScenarioStep::executeAndReturnResult);
+    return getScenarioSteps().stream()
```

Comment:
what does it do?

Reply:
ohh I see, sounds okey (maybe `.parallelStream()` might be a solution as well?)

Reply:
I think that scenario `steps` implies it is done serially and I think some tests may rely in this property. In general it could be unsafe to run some further step if previous failed, but if they turn out to be independent it is better to know that they are ok. Also not sure if they will be any faster in parallel as bazel seems to tell other invocations to wait for the current one to finish.